docs/UserGuide.md

```diff
@@ -345,6 +405,7 @@ Team size: 5
 
 1. **Validity of date entered:** The current behaviour of the `schedule` command does not check for the validity of the date entered. We plan to implement a check to ensure that the date entered is valid.
 2. **Future date check:** The current behaviour of the `schedule` command does not check if the valid date entered is in the future. We plan to implement a check to ensure that the date entered is in the future.
+3. **Validity of email domain entered:** The current behaviour of the `edit` command does not check for the validity of the domain name of the email entered. We plan to implement a check to ensure that a valid domain name is entered.
```

Comment:
It actually does check for validity.

![image](https://github.com/AY2324S2-CS2103T-T15-2/tp/assets/90739493/cb18dc10-3a08-4bbd-afc8-fdbb084d71dd)

Reply:
Oh what I actually meant is that we have yet to implement a checker to check for valid domain name for `email`. As shown here, editing the email to anything random (invalid) after the `@` is accepted. The one you mentioned above is only checking the validity of the local-part so yup I think is better to include the checking of domain name as future enhancement 

![Screenshot 2024-04-13 at 11 08 35 AM](https://github.com/AY2324S2-CS2103T-T15-2/tp/assets/122202483/7364b173-169c-4030-ab97-a85a8e51b084)

Reply:
Maybe I will specify it in the UG again to define what is the validity of the domain name of `email`.